boards/nordic/nrf54l20pdk/Kconfig.defconfig

```diff
@@ -6,9 +6,6 @@ if BOARD_NRF54L20PDK_NRF54L20_CPUAPP
 config ROM_START_OFFSET
 	default 0x800 if BOOTLOADER_MCUBOOT
 
-config SOC_NRF54LX_SKIP_CLOCK_CONFIG
-	default y
```

Comment:
This Kconfig skips the write to `NRF_OSCILLATORS`:
https://github.com/zephyrproject-rtos/hal_nordic/blob/c563b0e56dfa7e3915637b12b3428e6de36012be/nrfx/mdk/system_nrf54l.c#L73
which may be necessary for certain targets - can you verify this works on L20 currently?

Reply:
Added L09 support and verified that hello_world works ok on L09, too.